source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -364,6 +364,12 @@ Your application should not trust any input, for example from a file or an
 API. So the application should handle all operational errors and recover from
 them.
 
+## Security
+
+> Maintain NodeJS security best practices.
+
+Your application should be kept up to date. The NodeJS page has a list of NodeJS Security best Practices that should be adhered to.
```

Comment:
This call to action doesn't link to the specific page, which isn't that helpful to the reader.

Are there any specific best practices that we have a GDS-specific review on? Are some of these recommendations things that all developers need to keep vigilant for in writing and reviewing code, and some that can be enforced in CI/CD pipeline?